setup.py

```diff
@@ -26,8 +27,12 @@ def get_cmake_version():
     return output.splitlines()[0].split()[2]
 
 
-class MultipyRuntimeDevelop(develop):
-    user_options = develop.user_options + [("cmakeoff", None, None)]
+class MultipyRuntimeCmake(object):
+    user_options = [("cmakeoff", None, None)]
+
+
+class MultipyRuntimeDevelop(MultipyRuntimeCmake, develop):
```

Comment:
Can you rename this to MultipyRuntimeOptions, I'm adding another option to handle ABI as its needed for the test in core.

Reply:
The suffixes are matched to the cmdclass keys (build/develop/install) . Are you adding a build extension for ABI? This class can be subclassed I think.